hack/update-staging-godeps-dockerized.sh

```diff
@@ -51,6 +51,7 @@ while getopts ":df" opt; do
 done
 
 kube::util::ensure_dockerized
+KUBE_GOPATH="/godeps"
```

Comment:
don't like to change global state. Can't we prepend this to the godep calls in this file?